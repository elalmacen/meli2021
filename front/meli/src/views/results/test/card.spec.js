import React from 'react';
import {shallow} from 'enzyme';
import View from '../card';

describe('test searchBox component', () => {
    test('renders with product', () => {
        const product = {
            id: 12,
            picture: "https://http2.mlstatic.com/D_NQ_NP_848472-MLA43108239303_082020-O.webp",
            free_shipping : 'price free',
            price : {
                amount: 20,
                decimals: 1
            },
            title: "batman",
            region: "Villa del parque"

        };
        const wrapper = shallow(<View product={product}/>);
        expect(wrapper).toMatchSnapshot();
    });    
});
