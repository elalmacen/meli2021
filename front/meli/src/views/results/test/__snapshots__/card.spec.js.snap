// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test searchBox component renders with product 1`] = `
<li>
  <Link
    to="/items/12"
  >
    <Row
      noGutters={false}
    >
      <Col
        md={2}
        xs={12}
      >
        <img
          alt="thumbnail"
          className="img-fluid picture"
          height={180}
          src="https://http2.mlstatic.com/D_NQ_NP_848472-MLA43108239303_082020-O.webp"
          width={180}
        />
      </Col>
      <Col
        md={7}
        xs={12}
      >
        <p
          className="price free"
        >
          <CurrencyFormat
            allowNegative={true}
            decimalSeparator=","
            displayType="text"
            fixedDecimalScale={false}
            isAllowed={[Function]}
            isNumericString={false}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseUp={[Function]}
            onValueChange={[Function]}
            prefix="$ "
            suffix=""
            thousandSeparator="."
            thousandSpacing="3"
            type="text"
            value={20}
          />
          <span
            className="decimals"
          >
            1
          </span>
        </p>
        <h2>
          batman
        </h2>
      </Col>
      <Col
        md={3}
        xs={12}
      >
        <span>
          Villa del parque
        </span>
      </Col>
    </Row>
  </Link>
</li>
`;
